frontend/packages/operator-lifecycle-manager/integration-tests-cypress/views/catalog-source.view.ts

```diff
@@ -0,0 +1,57 @@
+import { modal } from '../../../integration-tests-cypress/views/modal';
+import { listPage } from '../../../integration-tests-cypress/views/list-page';
+
+const verifyPackageManifest = (opName: string, csName: string, depth: number = 0) => {
+  // prevent infinite recursion of verifyPackageManifest
+  expect(depth).toBeLessThan(20);
+  cy.log('verify the PackageManifest has been created');
+
+  return cy
+    .visit(
+      `/k8s/ns/openshift-marketplace/packages.operators.coreos.com~v1~PackageManifest?packagemanifest-name=${opName}`,
+    )
+    .get(`[data-test-rows="resource-row"]`)
+    .should('exist')
+    .then(() => {
+      // the opName can exist in multiple CatalogSources, so check for the csName
```

Comment:
We could use the new "Operators" tab on the CatalogSource added in #7430. I'd think it would simplify the logic here and add more test coverage for that feature.

Reply:
Good suggestion. @spadgett.  I've implemented it.  Please take another look.